library/src/test/kotlin/wu/seal/jsontokotlin/JsonToKotlinBuilderTest.kt

```diff
@@ -1,12 +1,101 @@
 package wu.seal.jsontokotlin
 
 import com.winterbe.expekt.should
-import org.junit.Before
 import org.junit.Test
 
 class JsonToKotlinBuilderTest {
 
 
+    @Test
+    fun fullMethodTest() {
+        val json1 = """{ "programmers": [
+                { "isFirstName": "Brett", "lastName":"McLaughlin", "email": "aaaa" },
+                { "firstName": "Jason", "lastName":"Hunter", "email": "bbbb" },
+                { "firstName": "Elliotte", "lastName":"Harold", "email": "cccc" }
+                ],
+                "authors": [
+                { "firstName": null, "lastName": "Asimov", "genre": "science fiction" },
+                { "firstName": "Tad", "lastName": "Williams", "genre": "fantasy" },
+                { "firstName": "Frank", "lastName": "Peretti", "genre": "christian fiction" }
+                ],
+                "musicians": [
+                { "firstName": "Eric", "lastName": "Clapton", "instrument": "guitar" },
+                { "firstName": "Sergei", "lastName": "Rachmaninoff", "instrument": "piano" }
+                ] } """
+
+        val actualOutput = JsonToKotlinBuilder()
+                .setPackageName("com.my.package.name")
+                .enableVarProperties(false) // optional, default : false
+                .setPropertyTypeStrategy(PropertyTypeStrategy.AutoDeterMineNullableOrNot) // optional, default :  PropertyTypeStrategy.NotNullable
+                .setDefaultValueStrategy(DefaultValueStrategy.AvoidNull) // optional, default : DefaultValueStrategy.AvoidNull
+                .setAnnotationLib(TargetJsonConverter.MoshiCodeGen) // optional, default: TargetJsonConverter.None
+                .enableComments(true) // optional, default : false
+                .enableOrderByAlphabetic(true) // optional : default : false
+                .enableInnerClassModel(true) // optional, default : false
+                .enabelMapType(true)// optional, default : false
+                .enableCreateAnnotationOnlyWhenNeeded(true) // optional, default : false
+                .setIndent(4)// optional, default : 4
+                .setParentClassTemplate("android.os.Parcelable") // optional, default : ""
+                .enableKeepAnnotationOnClass(true) // optional, default : false
```

Comment:
The two keepAnotation should only enable one of them, not all of them, for they are Conflict each other

Reply:
Okay